lib/commands/gesture.js

```diff
@@ -452,6 +456,12 @@ helpers.mobileScroll = async function (opts={}) {
   }
 };
 
+commands.scrollToElement = async function (element) {
```

Comment:
The support of UIClient-based automation is going to be deprecated. The main iOS automation driver is XCTest

Reply:
We know, after discussing this with Jonathan (and later Isaac) we decided we'll do the contribution for UIAutomation anyway, and XCUI will probably follow soon.